osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -198,10 +200,10 @@ protected override void LogoArriving(OsuLogo logo, bool resuming)
 
             bool displayLogin()
             {
-                if (!loginDisplayed)
+                if (!statics.Get<bool>(Statics.LoginOverlayDisplayed))
```

Comment:
i think it'd be cleaner to use the BDL load method to retrieve a bindable only, rather than querying `statics` multiple times.